app/models/concerns/placement_query.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PlacementQuery
```

Comment:
I think on balance this would be better as a query class in `app/services/placements`, invoked as `GetTraineesToBePlaced.call(provider)`. That feels a bit more explicit and less magic than having it as a concern. Then the reference to `trainees` below will become `provider.trainees` but otherwise no meaningful change. Whatchu reckon?

Reply:
how about as an instance method under provider?